mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/ids/BlobId.java

```diff
@@ -30,7 +30,7 @@
 public class BlobId {
     public static BlobId forPayload(byte[] payload) {
         Preconditions.checkArgument(payload != null);
-        return new BlobId(DigestUtils.sha1Hex(payload));
+        return new BlobId(DigestUtils.sha256Hex(payload));
```

Comment:
On this PR you have to make three changes for changing the hash used.
I think in a perfect word we should have a static method called "hashForFile" for example that hide the choosen implementation.